update billing screenshot (#3502)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Accounts/delete-users.md

1. From dbt Cloud, click the gear icon at the top right and select **Account Settings**.

> <Lightbox src="/img/docs/dbt-cloud/Navigate To Account Settings.png" width="85%"  title="Navigate to account settings" />

Reviewer: note: same thought as before regarding figure captions. not flagging/not adding a suggestion since you only updated the size on this line and not the text

Author: ok yea i just wanted to change the sizing because it was too small. regarding the title - it looks like this was inherited from someone's previous [pr](https://github.com/dbt-labs/docs.getdbt.com/pull/2535/files#diff-9fb77076c9811e949b92518289122086c45eebc710e381bc4aa9009e01e96a41) that we all touched at some point.

2. In **Account Settings**, select **Users** under **Teams**.
3. Select the user you want to delete, then click **Edit**. 
4. Click **Delete** in the bottom left. Click **Confirm Delete** to immediately delete the user without additional password prompts. This action cannot be undone. However, you can re-invite the user with the same information if the deletion was made in error. 

<Lightbox src="/img/docs/dbt-cloud/delete_user_20221023.gif" width="85%" title="Deleting a user" />

If you are on a **Teams** plan and you are deleting users to reduce the number of billable seats, you also need to take these steps to lower the license count:
1. In **Account Settings**, select **Billing**. 

> 2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. Leaving these blank won't allow you to save your changes.

Reviewer: ```suggestion
2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. If any field is left blank, you won't be able to save your changes.
```

Author: this sugg is passive voice so i'm switching it to active

> 2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. Leaving these blank won't allow you to save your changes.

Author: ```suggestion
2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. If you leave any field blank, you won't be able to save your changes.
```

> 3. Press **Update Payment Information** to save your changes. 

Reviewer: ```suggestion
3. Click **Update Payment Information** to save your changes. 
```

2. Enter the number of developer seats you want and click **Save**.
2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. Leaving these blank won't allow you to save your changes.
3. Press **Update Payment Information** to save your changes. 

> <Lightbox src="/img/docs/dbt-cloud/change-developer-seats.png" title="Developer seats" />

Reviewer: is this PNG being used in other pages? if not, suggest deleting it to keep our `/img` folder clean

Author: <img width="992" alt="Screenshot 2023-06-12 at 13 52 10" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/4e1229c6-dd65-429e-9259-7b2f5102e6f7">
